apps/ucp-ui/env/staging.env

```diff
@@ -2,3 +2,4 @@ LAUNCH_DARKLY_CLIENT_ID=66e06a7e7b99e2105cfe099d
 AUTH0_DOMAIN=auth-staging.universalconnectproject.org
 AUTH0_CLIENT_ID=osS8CuafkPsJlfz5mfKRgYH942Pmwpxd
 IS_STAGING=true
+WIDGET_DEMO_BASE_URL=https://demo-staging.widget.universalconnectproject.org
```

Comment:
We ran into an issue in the past because the env file didn't have a new line. Some of the github actions attempt to append variables to these files and if there isn't a new line it creates a problem like this:

```
WIDGET_DEMO_BASE_URL=https://demo-staging.widget.universalconnectproject.orgOTHER_ENV_VAR=otherThing
```